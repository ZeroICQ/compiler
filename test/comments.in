//abc
// abc// abs
///abc
10//abc
 20// abc
 30/ /40//
