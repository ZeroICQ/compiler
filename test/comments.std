(4:0)	integer literal	[10]	"10"
(5:1)	integer literal	[20]	"20"
(6:1)	integer literal	[30]	"30"
(6:3)	operator	[/]	"/"
(6:5)	operator	[/]	"/"
(6:6)	integer literal	[40]	"40"
(6:10)	EOF	[end of file]	"EOF"
