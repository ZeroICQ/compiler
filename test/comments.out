(1:0)	comment	[]	"//abc"
(2:0)	comment	[]	"// abc// abs"
(3:0)	comment	[]	"///abc"
(4:0)	integer literal	[10]	"10"
(4:2)	comment	[]	"//abc"
(5:1)	integer literal	[20]	"20"
(5:3)	comment	[]	"// abc"
(6:1)	integer literal	[30]	"30"
(6:3)	operator	[SLASH]	"/"
(6:5)	operator	[SLASH]	"/"
(6:6)	integer literal	[40]	"40"
(6:8)	comment	[]	"//"
(6:10)	EOF	[]	" "
